packages/redux/src/raffles/types/state.types.ts

```diff
@@ -0,0 +1,9 @@
+import type { BlackoutError, Raffle } from '@farfetch/blackout-client/src';
+import type { CombinedState } from 'redux';
+
+export type RafflesState = CombinedState<{
+  error: BlackoutError | null;
+  isLoading: boolean;
+  result: { entries: Array<Raffle['id']>; nextToken: string } | null;
```

Comment:
Please, create another type that extends from the `Raffles` type like this, since the `nextToken` property is inherited from `Raffles` type:

```js
export type RafflesNormalized = Omit<Raffles, 'entries'> & {
   entries: Array<Raffle['id']>;
}
```
Then you can define the state like this:

```suggestion
  result: RafflesNormalized | null;
```